File: search_linear/sample.swift
// http://swiftstub.com/ if you don't have a mac

var a = [1,2,3,4,6];
var searchValue = 3;

func search(array:[Int], searchValue:Int) -> Int {
    
    var result = -1;
    
    for var i = 0 ; i < array.count ; i++ {
        if array[i] == searchValue {
         result = i;
        }
    }
    return result;
    
}

var result = search(a,searchValue);

if result > -1 {
    // if you want it on string convert it to string or 
    var message = String(searchValue)+" found in index "+String(result);
    println(message);
}else{
    // you way use it this way also
    println("\(searchValue) not found in array");
}

File: search_linear/sample.js
var a = [15, 23, 24, 32, 7],
	searchValue = 24;

var search = function(searchArray,searchValue){
	var index = -1;
	for(var i = 0; i < searchArray.length; i++){
		if(searchArray[i] == searchValue){
			index = i;
			break;
		}
	}
	return index;
}

var result = search(a,searchValue);

if(result > -1){
	console.log(searchValue+" has been found at "+result);
}else{
	console.log(searchValue+" not found");
}